docs/templates/topindex.html

```diff
@@ -9,43 +9,18 @@ <h1>MicroPython documentation</h1>
   </p>
 
   <p>
-    MicroPython runs on a variety of systems and each has their own specific
-    documentation.  You are currently viewing the documentation for
-    <strong>{{ port_name }}</strong>.
+    MicroPython runs on a variety of systems and hardware platforms.  Here you can read
```

Comment:
Should link to https://micropython.org/resources/docs/unified-docs/reference/glossary.html#term-micropython-port somewhere here too.

Reply:
Done.